src/api_server/src/parsed_request.rs

```diff
@@ -22,13 +22,17 @@ use ApiServer;
 
 use vmm::rpc_interface::{VmmAction, VmmActionError};
 
-#[allow(clippy::large_enum_variant)]
+/// Helper function to avoid boiler-plate code.
```

Comment:
Since this is used in other files, I think it would look better as a static method `ParsedRequest::sync`

Reply:
done